assets/dev/js/editor/components/validator/breakpoint.js

```diff
@@ -0,0 +1,55 @@
+const NumberValidator = require( 'elementor-validator/number' );
+
+module.exports = NumberValidator.extend( {
+	breakpointKeysArray: [],
+
+	getDefaultSettings: function() {
+		return {
+			validationTerms: {
+				// Min width we allow for mobile
+				min: 320,
```

Comment:
You don't need the min definition since we added min value for mobile

Reply:
Done.